tests/validate_schemas.py

```diff
@@ -0,0 +1,57 @@
+import json
+import pathlib
+
+from jsonschema.validators import Draft7Validator
+
+
+HERE = pathlib.Path(__file__).parent
+SCHEMA_SRC = HERE / ".." / "format-specs" / "schema.json"
+SCHEMA = json.loads(SCHEMA_SRC.read_text())
+
+# use pytest?
+
+def test_valid_schema(path):
+
+    with open(path, "r") as f:
+        metadata = json.load(f)["geo"]
+
+    print(f"Validating {path}")
+    
+    errors = Draft7Validator(SCHEMA).iter_errors(metadata)
+
+    valid = True
+    for error in errors:
+        valid = False
+        print(f"  - {error.json_path}: {error.message}", "warning")
+        if "description" in error.schema:
+            print(f"    \"{error.schema['description']}\"", "warning")
+
+    if valid:
+        print("This is a valid GeoParquet file.\n", "success")
+    else:
+        print("This is an invalid GeoParquet file.\n", "error")
+        exit(1)
```

Comment:
I started to write this file based on the existing code we had for `geoparquet_validator/__init__.py`, but if just focusing on testing those json files against the schema, it could probably be simplified. Using something like pytest might actually made it easier to have clearer output if it fails.

Reply:
I'm +1 on using pytest